packages/components/src/__actions__/Menu/v3/Menu.module.scss

```
@@ -0,0 +1,38 @@
+@import "~@kaizen/design-tokens/sass/border";
+@import "~@kaizen/design-tokens/sass/color";
+@import "~@kaizen/design-tokens/sass/shadow";
+@import "~@kaizen/design-tokens/sass/spacing";
+@import "~@kaizen/design-tokens/sass/typography";
+
+.menu {
+  background-color: $color-white;
+  color: $color-purple-800;
+  width: 248px;
+  max-height: 22rem;
+  overflow: auto;
+  padding-block: $spacing-xs;
+  outline: none;
+  border-radius: $border-solid-border-radius;
+  box-shadow: $shadow-large-box-shadow;
+}
+
+.menu :global(.react-aria-Header) {
+  font-family: $typography-heading-6-font-family;
+  font-size: $typography-heading-6-font-size;
+  letter-spacing: $typography-heading-6-letter-spacing;
+  font-weight: $typography-heading-6-font-weight;
+  line-height: $typography-heading-6-line-height;
+  padding: $spacing-6 10px;
+  margin-inline: $spacing-xs;
+}
+
+.menu section:not(:last-of-type) {
+  &::after {
+    width: 100%;
+    height: 1px;
+    background-color: $border-solid-border-color;
+    content: "";
+    display: block;
+    margin-block: $spacing-xs;
```

Part of my ruthless campaign to eradicate t-shirt sizes ;)

```suggestion
    margin-block: $spacing-6;
```